src/types/date.ts

```diff
@@ -1,9 +1,10 @@
-import {STRules} from '../rules';
 import {StrongType, makeStrong} from '../strong-type';
 
+import {STRules} from '../rules';
+
 export type StrongDate = StrongType<string>;
 
-export function makeDate(fallback: string, initial?: string | null): StrongDate {
+export function makeDate(fallback: string, initial: string | null | undefined): StrongDate {
```

Comment:
- [ ] Change `| undefined` type on `initial` to use the optional operator instead `?`.